File: examples/hello-todo-fastapi/src/hello_todo_fastapi/api/health.py
"""Health check router."""

from fastapi import APIRouter

router = APIRouter()


@router.get("/healthz")
async def health() -> dict[str, str]:
    """Return service health status.

    Does not check storage health — the in-memory store is always available
    while the process is running.
    """
    return {"status": "ok"}

File: templates/skeletons/go-gin-layered/internal/service/health.go
package service

// HealthService implements health-check application logic.
type HealthService struct{}

// NewHealthService returns a new HealthService.
func NewHealthService() *HealthService {
	return &HealthService{}
}

// Status returns the current health status string.
func (s *HealthService) Status() string {
	return "ok"
}
